client/src/Register.css

```diff
@@ -0,0 +1,69 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap');
```

Comment:
i don't think we need both these imports, right?

Reply:
We needed them :/ Bianca helped us with this because the font wouldn't turn out without the imports

Reply:
Got it, it seems like we already have the imports in `App.css` though? So these imports would be redundant, as long as you change your `App.css` import (see my reply to Ranon's comment).